app/Actions/DashboardActions.js

```diff
@@ -1,3 +1,5 @@
+'use strict';
```

Comment:
Because if this [comment](https://github.com/marmelab/react-admin/pull/28#discussion_r31432574)